cmake/Modules/FindZstd_EP.cmake

```diff
@@ -98,6 +98,7 @@ if (NOT ZSTD_FOUND)
         -DCMAKE_BUILD_TYPE=Release
         -DCMAKE_INSTALL_PREFIX=${TILEDB_EP_INSTALL_PREFIX}
         -DCMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}
+	-DZSTD_BUILD_SHARED=OFF
```

Comment:
Was this required to get something to work? Could use a comment for the future, at least in the squashed commit message.

Reply:
Commented added.

Reply:
🙌